projects/1.21-fabric/assets/modrinth-trickster/trickster/lang/zh_cn.json

```
@@ -0,0 +1,288 @@
+  "trickster.widget.enter_scroll_name": "输入卷轴名：",
+  "trickster.widget.scroll_sign_note.line0": "注意！在你署名后，",
+  "trickster.widget.scroll_sign_note.line1": "它将不能再被修改。",
+  "trickster.widget.load_button": "写入卷轴",
+  "trickster.message.missing_scroll": "另一只手持有卷轴与笔以导入示例法术",
+  "trickster.message.immutable_scroll": "无效卷轴：无法写入",
+  "trickster.message.loaded_example": "示例已导入！",
+  "trickster.message.import_example": "是否要导入此示例法术？",
+  "trickster.message.import_example.description": "这么做会将另一只手中卷轴与笔上的法术§l覆盖§r为该示例法术。",
+  "trickster.item_group": "戏法师",
+  "trickster.scroll_hat": "所选槽位：%d",
+  "trickster.scroll_executable": "单次使用（%f 千甘道夫）",
+  "trickster.unnamed_entity": "未命名的%s",
+  "trickster.trick.trickster.add": "吞并之谋略",
+  "trickster.trick.trickster.subtract": "逃亡之谋略",
+  "trickster.trick.trickster.multiply": "统治之谋略",
+  "trickster.trick.trickster.divide": "屈从之谋略",
+  "trickster.trick.trickster.modulo": "整体之曲真",
+  "trickster.trick.trickster.power": "指数之曲真",
+  "trickster.trick.trickster.max": "高贵之谋略",
+  "trickster.trick.trickster.min": "低位之谋略",
+  "trickster.trick.trickster.ceil": "壮丽之曲真",
+  "trickster.trick.trickster.floor": "谦卑之曲真",
+  "trickster.trick.trickster.round": "客观之曲真",
+  "trickster.trick.trickster.sin": "甲型几何之曲真",
+  "trickster.trick.trickster.cos": "乙型几何之曲真",
+  "trickster.trick.trickster.tan": "丙型几何之曲真",
+  "trickster.trick.trickster.arcsin": "甲型反几何之曲真",
+  "trickster.trick.trickster.arccos": "乙型反几何之曲真",
+  "trickster.trick.trickster.arctan": "丙型反几何之曲真",
+  "trickster.trick.trickster.arctan2": "笛卡尔角之曲真",
+  "trickster.trick.trickster.if_else": "决策之曲真",
+  "trickster.trick.trickster.equals": "对抵之谋略",
+  "trickster.trick.trickster.not_equals": "失抵之谋略",
+  "trickster.trick.trickster.all": "无例外之谋略",
+  "trickster.trick.trickster.any": "通常态之谋略",
+  "trickster.trick.trickster.none": "缺失态之谋略",
+  "trickster.trick.trickster.extract_x": "甲元之曲真",
+  "trickster.trick.trickster.extract_y": "乙元之曲真",
+  "trickster.trick.trickster.extract_z": "丙元之曲真",
+  "trickster.trick.trickster.merge_vector": "吸收之曲真",
+  "trickster.trick.trickster.dot_product": "合向之曲真",
+  "trickster.trick.trickster.cross_product": "垂直之曲真",
+  "trickster.trick.trickster.normalize": "正规之曲真",
+  "trickster.trick.trickster.align_vector": "合向正规之曲真",
+  "trickster.trick.trickster.invert": "取反之曲真",
+  "trickster.trick.trickster.length": "辐长之曲真",
+  "trickster.trick.trickster.two": "基础之错觉",
+  "trickster.trick.trickster.read_spell": "记事员之错觉",
+  "trickster.trick.trickster.write_spell": "记事员之技巧",
+  "trickster.trick.trickster.write_closed_spell": "禁文记事员之技巧",
+  "trickster.trick.trickster.write_macro_ring": "宏之技巧",
+  "trickster.trick.trickster.read_macro_ring": "宏之错觉",
+  "trickster.trick.trickster.reveal": "展示之谋略",
+  "trickster.trick.trickster.read_crow_mind": "乌鸦灵思之错觉",
+  "trickster.trick.trickster.write_crow_mind": "乌鸦灵思之技巧",
+  "trickster.trick.trickster.cost": "消耗之技巧",
+  "trickster.trick.trickster.reflection": "定位之错觉",
+  "trickster.trick.trickster.caster_reflection": "精思之错觉",
```

~hexcasting译上头了是吧~
```suggestion
  "trickster.trick.trickster.caster_reflection": "自反之错觉",
```
或者其他之类的，我觉得要把“自”体现出来（

---

~~致敬的体现~~
需要改吗

---

精思完全看不出跟自己有关系
reflection理论上可以从反省->自身嘛（